components/x-topic-search/readme.md

```diff
@@ -1,6 +1,6 @@
 # x-topic-search
 
-This module provides a topic search bar.
+This module provides a topic search bar. It displays topic suggestions which are fetched from the api, passed as a property, based on user input(search term). x-follow-button is displayed with the topic suggested.
```

Comment:
Maybe this should be more focussed on the user and less on the technical detail of how it works?

>This module allows a user to search for topics by name, and follow them. If the an already-followed topic is return in the search results, then those topics are indicated as such.

Reply:
Agree with this actually. Might still be worth mentioning where it gets its suggestions from though.

Reply:
Thank you, @benbarnett and @dan-searle!